test/integration/disruption/disruption_test.go

```diff
@@ -163,6 +163,9 @@ func TestPDBWithScaleSubresource(t *testing.T) {
 	waitPDBStable(t, clientSet, 4, nsName, pdb.Name)
 
 	newPdb, err := clientSet.PolicyV1beta1().PodDisruptionBudgets(nsName).Get(pdb.Name, metav1.GetOptions{})
+	if err != nil {
+		t.Error(err)
```

Comment:
> t.Errorf("Error getting PodDisruptionBudget: %v", err)

makes it easier to search and to match the error handling of the `Create` call above (L159).